spec/classes/jira_plugin_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper.rb'
+
+describe 'jira' do
```

Comment:
Remove this level.

Also, this file should be named `spec/defines/jira_plugin_spec.rb`.